models/article-models.js

```diff
@@ -14,14 +14,43 @@ exports.articleById = (article_id) => {
     });
 };
 
-exports.returnArticles = () => {
-  return db
-    .query(
-      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC`
-    )
-    .then(({ rows }) => {
-      return rows;
-    });
+exports.returnArticles = (query) => {
+  const { sort_by, order } = query;
+
+  const allowedSortInputs = [
+    "title",
+    "topic",
+    "author",
+    "body",
+    "created_at",
+    "votes",
+  ];
+
+  if (sort_by && !allowedSortInputs.includes(sort_by)) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+
+  if (order && order !== "desc" && order !== "asc") {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+
```

Comment:
Nice, but do you want to be case sensitive for order? Or should someone be allowed to query for `/api/articles?sort_by=title&order=ASC`?